packages/address/src/AddressFormatter.ts

```diff
@@ -17,19 +23,18 @@ export default class AddressFormatter {
   }
 
   async getCountry(countryCode: string): Promise<Country> {
-    let country = this.loadCountryFromCache(countryCode);
```

Comment:
I partially rolled back. Now, it uses a `const` and avoids the useless `await`, but it's otherwise the same exact behavior in the same exact order/structure.

Reply:
Cool, this looks good to me